test_suite/package/otel_package/test_otel.py

```diff
@@ -1,14 +1,15 @@
 import time
 import pytest
 from lib import console_lib
+from test_suite.generic import helpers
```

Comment:
```suggestion
from test_suite import helpers
```
Now that I think about this, maybe we should move the helpers outside of the "generic" dir?
The "generic" dir name actually refers to the fact that test cases inside there may apply to most/all cloud providers.

Reply:
I don't see an issue with this being inside the generic dir, in fact the function we moved there from the first place "__get_instance_data_from_json" is being used by "instance_data" and not only for aws.
If we see more general functions in the "helpers" file that are meant to be only for one cloud provider, maybe we should consider separating this.